kore/src/Kore/Parser/Parser.hs

```diff
@@ -411,21 +388,21 @@ BNF definitions:
 ⟨set-variable⟩ ::= ⟨set-variable-identifier⟩ ‘:’ ⟨sort⟩
 @
 
-The @set-@ version always starts with @#@, while the regular one does not.
+The @set-@ version always starts with @\@@, while the regular one does not.
 -}
 variableParser :: Parser Variable
 variableParser = idParser >>= variableRemainderParser
 
 singletonVariableParser :: Parser Variable
```

Comment:
I know it wasn't introduced in this pull request, but doesn't _The Semantics of K_ use the terminology, "singular variable," not "singleton variable"?